<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ananymous chat</string>
  </property>
  <widget class="QGraphicsView" name="profile_photo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="data_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Some data</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chats_cbox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="Chats_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your chats (1 onlne)</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="new_chat_button">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Start new chat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edit_profile_button">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Edit profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
